lib/utils/networkUtils.js

```diff
@@ -2,22 +2,36 @@
  * @module Contains helpers functions for working with network requests
  */
 
-const fetch = require('node-fetch');
+const https = require('https');
 const fs = require('fs');
+const axios = require('axios');
+
+/** Used to send request to our (Bigcommerce) servers only.
+ *  Shouldn't be used to send requests to third party servers because we disable https checks
+ *
+ * @param {object} options
+ * @returns {Promise<object>}
+ */
+async function sendApiRequest(options) {
+    return axios({
+        httpsAgent: new https.Agent({ rejectUnauthorized: false }),
```

Comment:
Do we really need to create a new agent for each request?

Reply:
Maybe we can reuse it, kinda:
```
const httpsAgent = new https.Agent({ rejectUnauthorized: false });
axios({ httpsAgent, ...options });
```
But need to check if it doesn’t cache or reuse something leading to unexpected behavior.

Reply:
Updated in the new commit